Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
2016-04-07 16:55 GMT+02:00 Michael Biebl <notifications@github.com>:

> In Makefile.am
> <https://github.com/rsyslog/libfastjson/pull/47#discussion_r58885504>:
>
> > @@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
> >
> >  libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
> >  	-export-symbols-regex '^fjson_.*'
> > -libfastjson_la_CPPFLAGS = -Werror
>
> Why does removing Werror from libfastjson_la_CPPFLAGS break AC_CHECK_FUNCS?
> I can understand that adding -Werror via CFLAGS can have an influence on
> AC_CHECK_FUNCS, but dropping it should be safe.
>
>
At this point, my main concern is travis and not to re-open old problems. I
have now managed to get travis require warning-free builds, I would like to
keep this while we improve the system.